Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -5414,6 +5491,30 @@ bool RenderLayer::isVisuallyNonEmpty(PaintedContentRequest* request) const
     return hasNonEmptyChildRenderers(*request);
 }
 
+RenderSVGResourceClipper* RenderLayer::svgClipperFromStyle() const
```

Comment:
The code shouldn't live in RenderLayer, it doesn't need any RenderLayer related information. How about moving it to RenderLayerModelObject?

Reply:
Good point, let's leave it as-is and decide later.